src/tests/dbus-tests/test_loop.py

```diff
@@ -0,0 +1,99 @@
+#!/bin/python3
+
+import unittest
+import storagedtestcase
+import subprocess
+import time
+import dbus
+import glob
+import os
+
+class StoragedLoopDeviceTest(storagedtestcase.StoragedTestCase):
+    """This is SwapSpace related functions unit test"""
+
+    LOOP_DEVICE_FILENAME = 'loop_device.img'
+
+    def get_device(self, dev_name):
```

Comment:
Not sure if this method is really needed. If you think so, you should probably add it to `StoragedTestCase`.